lib/Doctrine/Persistence/Mapping/MappingException.php

```diff
@@ -28,12 +28,16 @@ public static function classNotFoundInNamespaces($className, $namespaces)
     }
 
     /**
+     * @param class-string $driverClassName
+     *
      * @return self
      */
-    public static function pathRequired()
+    public static function pathRequired(string $driverClassName)
```

Comment:
This is a BC break. Shall we add a new named constructor (e.g. `pathRequiredForDriver`) instead?